app/components/TaskList.tsx

```diff
@@ -14,14 +14,79 @@ interface Props {
   projectRevision: TaskList_projectRevision$key;
 }
 
+const getStatus = (
+  forms: Array<{
+    changeStatus: string;
+    isPristine: boolean;
+    validationErrors: any[];
+  }>
+) => {
+  if (
+    forms.some((form) => form?.changeStatus === "pending" && !form.isPristine)
+  )
+    return "Incomplete";
+
+  if (
+    forms.some(
+      (form) =>
+        form.changeStatus === "staged" && form.validationErrors.length > 0
+    )
+  )
+    return "Attention Required";
```

Comment:
```suggestion
    return <strong>Attention Required</strong>;
```

this will remove the need for conditional rendering below